ProcessMessage/handler.ts

```diff
@@ -513,6 +513,18 @@ export const getProcessMessageHandler = ({
             createdMessageEvent.message.id
           )(MessageStatusValueEnum.PROCESSED)();
 
+          telemetryClient.trackEvent({
+            name: "api.messages.processed",
+            properties: {
+              fiscalCode: toHash(profile.fiscalCode),
+              messageId: createdMessageEvent.message.id,
+              mode: profile.servicePreferencesSettings.mode,
+              senderId: createdMessageEvent.message.senderServiceId,
+              timeFromCreated: Date.now() - newMessageWithoutContent.createdAt.getTime()
+            },
+            tagOverrides: { samplingEnabled: "false" }
```

Comment:
Since we'll treat this event for statistical analysis, maybe disabling sampling it's not necessary (but I see it's disabled in creation event too)